src/Sarif.Viewer.VisualStudio.Core/ResultTextMarker.cs

```diff
@@ -389,13 +402,21 @@ private bool TryToFullyPopulateRegionAndFilePath()
                 return false;
             }
 
-            if (File.Exists(this.resolvedFullFilePath) &&
+            if (this.fileSystem.FileExists(this.resolvedFullFilePath) &&
                 Uri.TryCreate(this.resolvedFullFilePath, UriKind.Absolute, out Uri uri))
             {
                 // Fill out the region's properties
                 this.fullyPopulatedRegion = FileRegionsCache.Instance.PopulateTextRegionProperties(this.region, uri, populateSnippet: true);
             }
 
+            if (this.fileSystem.FileExists(this.resolvedFullFilePath) &&
```

Comment:
>if [](http://example.com/codeflow?start=12&length=2)

Should this be an "else if" to skip it if the earlier "if" has run?

Reply:
yes combined into 1 "if" statement